receiver/oracledbreceiver/metadata.yaml

```diff
@@ -0,0 +1,176 @@
+    unit: "{parses}"
+  oracledb.pga_memory:
+    description: Session PGA (Program Global Area) memory
+    enabled: true
+    sum:
+      aggregation: cumulative
+      monotonic: true
+      value_type: int
+    unit: By
+  oracledb.physical_reads:
+    description: Number of physical reads
+    enabled: true
+    sum:
+      aggregation: cumulative
+      monotonic: true
+      value_type: int
+    unit: "{reads}"
+  oracledb.user_commits:
+    description: Number of user commits. When a user commits a transaction, the redo
+      generated that reflects the changes made to database blocks must be written
+      to disk. Commits often represent the closest thing to a user transaction rate.
+    enabled: true
+    sum:
+      aggregation: cumulative
+      monotonic: true
+      value_type: int
+    unit: "{commits}"
+  oracledb.user_rollbacks:
+    description: Number of times users manually issue the ROLLBACK statement or an
+      error occurs during a user's transactions
+    enabled: true
+    sum:
+      aggregation: cumulative
+      monotonic: true
+      value_type: int
+    unit: 1
+  oracledb.sessions.usage:
+    attributes:
+      - db.oracle.session.type
+      - db.oracle.session.status
+    description: Count of active sessions.
+    enabled: true
+    gauge:
+      value_type: int
+    unit: "{sessions}"
+  oracledb.processes.usage:
+    description: Current count of active processes.
+    enabled: true
+    gauge:
+      value_type: int
+    unit: "{processes}"
+  oracledb.processes.limit:
+    description: Maximum limit of active processes.
+    enabled: true
+    gauge:
+      value_type: int
+    unit: "{processes}"
+  oracledb.sessions.limit:
+    description: Maximum limit of active sessions.
+    enabled: true
+    gauge:
+      value_type: int
+    unit: "{sessions}"
+  oracledb.enqueue_locks.usage:
+    description: Current count of active enqueue locks.
+    enabled: true
+    gauge:
+      value_type: int
+    unit: "{locks}"
+  oracledb.enqueue_locks.limit:
+    description: Maximum limit of active enqueue locks.
+    enabled: true
+    gauge:
+      value_type: int
+    unit: "{locks}"
+  oracledb.enqueue_resources.usage:
+    description: Current count of active enqueue resources.
+    enabled: true
+    gauge:
+      value_type: int
+    unit: "{resources}"
+  oracledb.enqueue_resources.limit:
+    description: Maximum limit of active enqueue resources.
+    enabled: true
+    gauge:
+      value_type: int
+    unit: "{resources}"
+  oracledb.tablespace_size.limit:
+    attributes:
+      - db.oracle.tablespace.name
+    description: Maximum size of tablespace in bytes.
+    enabled: true
+    gauge:
+      value_type: int
+    unit: By
+  oracledb.tablespace_size.usage:
+    attributes:
+      - db.oracle.tablespace.name
+    description: Used tablespace in bytes.
+    enabled: true
+    gauge:
+      value_type: int
+    unit: By
+name: oracledbreceiver
+resource_attributes:
+  db.oracle.instance.name:
```

Comment:
Let's call it `oracledb.instance.name` for consistency with metric names. I don't see `db.` prefix in any of the existing scrapers